src/components/AddUser.js

```diff
@@ -1,25 +1,28 @@
 import React from "react";
+import { Button, Form } from 'semantic-ui-react';
 
 const AddUser = ({ onAdd }) => {
 
-const handleOnSubmit = (e) => {
+  const handleOnSubmit = (e) => {
     e.preventDefault();
-    onAdd(e.target.name.value,e.target.email.value);
+    onAdd(e.target.name.value, e.target.email.value);
     e.target.name.value = "";
     e.target.email.value = "";
-}
-
+  }
   return (
-    <div>
-      <form onSubmit={handleOnSubmit}>
-        <h3>Add User</h3>
-        <input placeholder="Name" name="name" />
-        <input placeholder="Email" name="email" />
-        <button onSubmit={handleOnSubmit}>Add</button>
-        <hr />
-      </form>
-    </div>
+    <Form onSubmit={handleOnSubmit}>
+      <Form.Field>
+        <label> Name</label>
+        <input placeholder="Name" name="name" required />
```

Comment:
In React, using uncontrolled components is a bad practice unless really necessary!
Replace them this controlled ones here!